README.md

```diff
@@ -9,194 +9,192 @@
-
-See https://fdc3.finos.org for more information, including on [compliance] and the [FDC3 charter], as well as a comprehensive [API Reference]. You can also take the free [FDC3 Training](https://www.edx.org/course/fdc3-interoperability-for-the-financial-desktop) for an introduction to FDC3's core concepts and usage.
-
-[FDC3 Charter]: https://fdc3.finos.org/docs/fdc3-charter
-[Compliance]: https://fdc3.finos.org/docs/fdc3-compliance
-[API Reference]: https://fdc3.finos.org/docs/api/ref/DesktopAgent
+- Users benefit from a more joined-up experience, which reduces the "friction" in getting common tasks done,
+- By enabling applications to:
+  - launch other apps (build a launcher),
+  - respond to activity in other apps (context sharing),
+  - request functionality from other apps (raising intents).
 
-# Supported Platforms
-
-As an open standard, FDC3 can be implemented on any platform and in any language.
-
-All that is required is a "desktop agent" that supports the FDC3 standard, which is responsible for co-ordinating application interactions.
+## What Are The Benefits?
 
-## Web
+### 📇 Help Manage Information Overload
 
-For web applications to be FDC3-enabled, they need to run in the context of an agent that makes the FDC3 API available to the application. This desktop agent is also responsible for launching and co-ordinating applications. It could be a browser extension, web app, or full-fledged desktop container framework.
+> Finance is an information-dense environment.  
+> Typically, traders will use serveral different displays so that they can keep track of multiple information sources at once.
+> FDC3 helps with this by sharing the "context" between multiple applications, so that they collectively track the topic the user is focused on.
 
-### Example
+### 🏃‍♂️ Work Faster
 
-```ts
-// declare FDC3-compliant data
-const instrument = {
-    type: 'fdc3.instrument',
-    id: {
-        ticker: 'AAPL',
-        ISIN: 'US0378331005',
-        FIGI : 'BBG000B9XRY4'
-    }
-}
+> FDC3 standardizes a way to call actions between applications (called "intents").
+> Applications can raise intents for other apps to resolve, extending each other's functionality.
+> Instead of the user copy-and-pasting bits of data from one application to another, FDC3 makes sure the intents have the data they need to seamlessly transition activity between applications.
 
-// invoke an action in another application, with the required data
-const result = await fdc3.raiseIntent('ViewAnalysis', instrument)
+### 🖥️  Platform Agnostic
 
-// join the red channel and broadcast data to subscribers
-await fdc3.joinChannel('red')
-fdc3.broadcast(instrument)
+> As an open standard, FDC3 can be implemented on any platform and in any language.
+> All that is required is a "Desktop Agent" that implements the FDC3 standard, which is responsible for co-ordinating application interactions.
```

Comment:
@robmoffat should we link here to https://fdc3.finos.org/community? Something like: 

"For a list of open source and proprietary desktop agents, see the "Platform providers" [here](https://fdc3.finos.org/community#type-platform-provider)"